services/manifest-repo-export-service/pkg/repository/transformer.go

```diff
@@ -21,24 +21,27 @@ import (
 	"database/sql"
 	"errors"
 	"fmt"
+	api "github.com/freiheit-com/kuberpult/pkg/api/v1"
+	"github.com/freiheit-com/kuberpult/pkg/auth"
 	"github.com/freiheit-com/kuberpult/pkg/db"
+	"github.com/freiheit-com/kuberpult/pkg/grpc"
+	"github.com/freiheit-com/kuberpult/pkg/logger"
+	billy "github.com/go-git/go-billy/v5"
+	"github.com/go-git/go-billy/v5/util"
+	yaml3 "gopkg.in/yaml.v3"
 	"io"
 	"os"
 	"strconv"
 	"strings"
-
-	"github.com/freiheit-com/kuberpult/pkg/logger"
-
-	yaml3 "gopkg.in/yaml.v3"
-
-	api "github.com/freiheit-com/kuberpult/pkg/api/v1"
-	"github.com/freiheit-com/kuberpult/pkg/auth"
-	billy "github.com/go-git/go-billy/v5"
-	"github.com/go-git/go-billy/v5/util"
+	"time"
 )
 
 const (
-	queueFileName = "queued_version"
+	queueFileName       = "queued_version"
+	fieldCreatedAt      = "created_at"
+	fieldcreatedByName  = "created_by_name"
```

Comment:
consistent naming please:
```suggestion
	fieldCreatedByName  = "created_by_name"
```